### PR TITLE
pt-BR: update some fuzzy translations

### PR DESCRIPTION
**Pull request by joaovicmendes**

### CODE REVIEW

**Comment by mgeisler** [comment]
Thanks @joaovicmendes! The PR looks great from a technical point of view :+1:

File: po/pt-BR.po [hunk 1]
```diff
@@ -3283,12 +3333,20 @@ msgid ""
...
-"    println!();"
+"    println!();\n"
+"\n"
+"    print!("Iterando sobre um intervalo:");\n"
+"    for i in 0..3 {\n"
+"        print!(" {}", array[i]);\n"
+"    }\n"
+"    println!();\n"
+"}\n"
+"```"
```
Comment (joaovicmendes) [inline po/pt-BR.po:3349]: My translation stopped here. The changes below came from `msgmerge`

**Comment by mgeisler** [comment]
Hi @rastringer and @hugojacob, please look this over when you get a minute.

File: po/pt-BR.po [hunk 2]
```diff
@@ -1469,29 +1446,26 @@ msgid ""
...
 "sees the `Android.bp` files in `src/android/`."
 msgstr ""
-"6. Se você não pular as partes específicas do Android no Dia 4, precisará de um [checkout\n"
-"   do AOSP][1]. Faça um checkout do [repositório do curso][2] no mesmo\n"
-"   computador e mova o diretório `src/android/` para a raiz do seu checkout doAOSP.\n"
-"   Isso garantirá que o sistema de compilação do Android veja os\n"
-"   arquivos `Android.bp` em `src/android/`."
+"Você precisará de um [checkout do AOSP][1]. Faça um checkout do [repositório\n"
+"do curso][2] no mesmo computador e mova o diretório `src/android/` para a raiz\n"
+"do seu checkout doAOSP. Isso garantirá que o sistema de compilação do Android veja os\n"
```
Comment (hugojacob) [inline po/pt-BR.po:1457]:
```suggestion
"do seu checkout do AOSP. Isso garantirá que o sistema de compilação do Android veja os\n"
```

File: po/pt-BR.po [hunk 3]
```diff
@@ -784,9 +776,8 @@ msgid "Driver"
 msgstr ""
 
 #: src/SUMMARY.md:239 src/SUMMARY.md:241
-#, fuzzy
 msgid "Using It"
-msgstr "Usando o Cargo"
+msgstr "Usando o driver"
```
Comment (hugojacob) [inline po/pt-BR.po:780]:
```suggestion
msgstr "Usando o Cargo"
```

Here "Cargo" refers to the package manager.

**Review by hugojacob — APPROVED** [review]

**Comment by joaovicmendes** [comment]
Hi, @hugojacob. I've updated the PR with the more recent changes to the repository, and I've fixed some syntax errors that were breaking the tests. Could you PTAL?

**Pull request merged**